RELEASING.md

```diff
@@ -108,7 +108,8 @@ Tagging the Release
 
 1. Verify there are no open issues in the release milestone. Open issues should
    either be deferred or resolved and the fix backported.
-2. For vMajor.Minor.x branch, change `README.md` to refer to the next release
+1. Continue with Google-internal steps at go/grpc/java/releasing.
+1. For vMajor.Minor.x branch, change `README.md` to refer to the next release
```

Comment:
I know you can use the `1` everywhere, but then it is ugly to read as a text file. Let's not change the style of this file.